wrappers/leveldb/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required (VERSION 3.10)
+
+include_directories(${PROJECT_SOURCE_DIR}/include)
+
+include(ExternalProject)
+ExternalProject_Add(
+  leveldb
+  GIT_REPOSITORY https://github.com/google/leveldb.git
+  GIT_TAG master 
```

Comment:
Whenever adding a third-party project, it is better to make it point to a stable release to avoid breaking the build randomly. You can replace `master` by the latest release of LevelDB.